### PR TITLE
fix(docs): update versions workflow

### PR DESCRIPTION
**Pull request by platon-rov**
## Related Issue(s)

Follows up https://github.com/SAP/fundamental-ngx/pull/8774

## Description

Fixes for update versions workflow.

Issues fixed
* workflow triggered with `workflow_run` event didn't get the commit that made in process of previous workflow.
* getting the version done wrong.

## Screenshots

No visual changes.

### CODE REVIEW

File: .github/actions/wait-for-netlify/index.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                clearInterval(handle);
+                return resolve();
+            }
+
+            console.log(`Not yet ready, waiting ${increment} more seconds...`);
+        }, increment * 1000);
+    });
+};
+
+const waitForUrl = async (url, MAX_TIMEOUT) => {
```
Comment (g-cheishvili) [inline .github/actions/wait-for-netlify/index.js:77]: We have library wait-on in our dependencies, you can use that instead

**Comment by g-cheishvili** [comment]
How can we test this?

**Comment by g-cheishvili** [comment]
Idea is cool with netlify deployment waits. Maybe we could use netlify instead of firebase for e2e tests? In the case of firebase, we are waiting for local build to finish. We could just wait for netlify results and that's all

File: .github/actions/wait-for-netlify/index.js (hunk 1 above)
Comment (platon-rov) [reply .github/actions/wait-for-netlify/index.js:77]: I copied `probablyup/wait-for-netlify-action` action and added the commit-sha input parameter as we need it, we probably can rewrite the whole thing in a better manner, but do we want to invest time in it? :)

**Comment by platon-rov** [comment]
> How can we test this?

Testing github actions is a tough deal but we can use `act` for it.

> Idea is cool with netlify deployment waits. Maybe we could use netlify instead of firebase for e2e tests? In the case of firebase, we are waiting for local build to finish. We could just wait for netlify results and that's all

https://github.com/SAP/fundamental-ngx/issues/8836

**Review by g-cheishvili — APPROVED** [review]

**Pull request merged**